external-scripts.json

```diff
@@ -1,6 +1,5 @@
 [
   "hubot-ascii-art",
-  "hubot-auth",
```

Comment:
This one had conflicts with the new `coffeescript` lib.  But I don't think we're using it for anything...

Reply:
Only thing I can imagine is to auth to Slack period.